### ISSUE TITLE
In Menu -> Format : Page settings and Reset shapes and positions have the same mnemonics letter

### ISSUE THREAD

**Issue by JLWALTENER** [musescore/MuseScore#23705]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1) Have a score open
2) Clic on Alt+O to open the Format menu
3) See at the menu : 'Page setting' and 'Reset shapes and positions' share the same mnemonics letter = P

Actual Behavior : Press Alt+O+P opens the Page settings dialog (Fortunately), but "Reset shapes and position" is not accessible via this mnemonics.

### Expected behavior

- For 'Format > Reset shapes and positions', make the mnemonic "H"
- For 'Format > Reset entire score to default layout', make the mnemonic "D"

### Supporting files, videos and screenshots

![MenuFormat Mnemonics1](https://github.com/user-attachments/assets/6a4e583f-8ee9-4691-b265-d74d5e0e2950)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 11 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.0-242030305, revision: github-musescore-musescore-cd25d06

### Regression

No.

### Operating system

Windows 11

### Additional context

Discovered during translation checking.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
@shoogle Quick thing: we just need to decide on a new mnemonic for "Reset shapes and positions" that isn't "P". I'd suggest "H". "R" is available, but since there are 4 different "Reset" options here, it doesn't seem to make sense to assign it to the third one in the list.

I also see that "Reset entire score to default layout" doesn't have a mnemonic. We could give that one "D".

Sound good?

**Comment by shoogle** [comment]
@avvvvve, I'm happy with 'H' for "Reset shapes and position" and 'D' for "Reset entire score to default layout".

**Comment by avvvvve** [comment]
Great! I updated the issue with that as the expected behavior. Thanks @JLWALTENER for the spot!
